killbill-integration-tests/core/test_account.rb

```diff
@@ -132,5 +132,32 @@ def test_account_update
 
     end
 
+    def test_account_missing_calls
```

Comment:
Instead of having one generic test, better to have a few _logical_ ones centered around a set of endpoints. It's easier for debugging tests failures (much better to have `test_account_blocking_state FAILED` in the tests reports rather than `test_account_missing_calls FAILED`, as the former tells you right away which piece of code has a regression).

For instance, we could have one `test_account_blocking_state`, which does:

* `account.blocking_states` and verify the returned list is empty
* `account.block_account` and verify response is success
* `account.blocking_states` and verify the returned list has now one element (and check the blocking state fields `state_name`, `service`, etc.)